htdocs/compta/facture/list.php

```diff
@@ -582,6 +582,8 @@
 $companyparent = new Societe($db);
 $company_url_list = array();
 $subtypearray = $object->getArrayOfInvoiceSubtypes(0);
+$soc = new Societe($db);
+$soc->fetch($socid);
```

Comment:
For the list $socid is most often empty.

Reply:
It's about the thirdparty own list
![1](https://github.com/Dolibarr/dolibarr/assets/93765174/14184fca-17db-48f0-aa1e-c145a84bc10d)

![2](https://github.com/Dolibarr/dolibarr/assets/93765174/295b56ea-f012-42c6-b561-b6ca162643cb)